src/main/java/commands/ListCommand.java

```diff
@@ -96,10 +99,15 @@ private String listChapters(Access access) {
         }
         result.append(String.format(MESSAGE_EXIST, CHAPTER));
         for (Chapter c : allChapters) {
+            if (c.getChapterName().equals("dues")) {
+                continue;
+            }
             if (allChapters.indexOf(c) == chapterCount - 1) {
-                result.append(allChapters.indexOf(c) + 1).append(".").append(c);
+                result.append(allChapters.indexOf(c) + 1).append(".")
+                        .append(c).append(" (due by ").append(c.getDueBy()).append(")");
```

Comment:
if the due date is null, you can show something like "No due date" instead of "due by null"